saleor/wishlist/models.py

```diff
@@ -0,0 +1,92 @@
+import uuid
+
+from django.db import models, transaction
+
+from ..account.models import User
+from ..product.models import Product, ProductVariant
+
+
+class WishlistQuerySet(models.QuerySet):
+    def get_or_create_wishlist_for_user(self, user):
+        try:
+            return user.wishlist
```

Comment:
Why not - `get_or_create`? https://docs.djangoproject.com/en/3.0/ref/models/querysets/